### PR TITLE
ci: Add a threshold for functional test pass

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Several tests are flaky because of the CI slowness. I've added a threshold value of 10% to still pass the suite in case of failure (currently allows 3 tests out of 30 to fail)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
cc @rerorero

**Review by rerorero — APPROVED** [review]
LGTM!

File: .github/workflows/functional-test.yml [hunk 1]
```diff
@@ -71,12 +74,7 @@ jobs:
     - uses: reactivecircus/android-emulator-runner@v2
       name: e2e_api${{ matrix.apiLevel }}
       with:
-        script: |
-          npm run e2e-test:driver
-          npm run e2e-test:commands
-          npm run e2e-test:commands:find
-          npm run e2e-test:commands:general
-          npm run e2e-test:commands:keyboard
+        script: bash -xe scripts/run_functional_tests.sh
```
Comment (KazuCocoa) [inline .github/workflows/functional-test.yml:77]:
```suggestion
        script: bash -xe scripts/run-functional-tests.sh
```
?

Comment (mykola-mokhnach) [reply .github/workflows/functional-test.yml]: good catch, fixed

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**